### ISSUE TITLE
Hyphens in application name?

### ISSUE THREAD

**Issue by mikelhamer** [aws-amplify/amplify-console#1494]
When using the Amplify Console, hyphens are allowed in the project name.

When using the Amplify CLI, hyphens are not allowed in the project name.

This seems like a major inconsistency that could lead to issues. Can someone chime in on this?

**Comment by Athena96** [comment]
Yes apologies, there are currently some naming inconsistencies between the Console and CLI. Marking this as a bug

**Comment by mikelhamer** [comment]
> 
> 
> Yes apologies, there are currently some naming inconsistencies between the Console and CLI. Marking this as a bug

Thanks! I'm wondering which one will be changed to match the other. I would very much prefer hyphens to be allowed.
